addrcmd.go

```diff
@@ -115,6 +120,28 @@ func filAddrs(cctx *cli.Context) error {
 			// If it's a masked 0x address, return the f0 and if enabled the expensive reverse lookup for the fX address
 			fmt.Println(addr)
 		}
+
+		if expensiveLookup {
```

Comment:
Not sure how frequently people are going to use this, my hope is if it's common enough someone will show up with a use case to determine how to handle it (e.g. if they're calling it repeatedly the type of cache that would be most useful). At the moment I don't do this more than occasionally for debugging so leaving it alone.